src/Microsoft.Azure.WebJobs.ServiceBus/ServiceBusAttribute.cs

```diff
@@ -58,5 +58,8 @@ public ServiceBusAttribute(string queueOrTopicName, AccessRights access)
         /// The default is "Manage".
         /// </summary>
         public AccessRights Access { get; private set; }
+
+        // Used to choose the correct entity type to create, if necessary
+        internal virtual bool IsQueue => true;
```

Comment:
I wonder if there isn't a SB entity type enum we can use here. Or perhaps we define our own.